concent_api/core/tests/test_core_views.py

```diff
@@ -148,7 +148,45 @@ def test_receive_return_http_204_if_no_messages_in_database(self):
         self.assertEqual(response.status_code, 204)  # pylint: disable=no-member
         self.assertEqual(len(MessageStatus.objects.filter(delivered=False)), 0)
 
-    def test_view_receive_out_of_band(self):
-        response = self.client.post(reverse('core:receive_out_of_band'), content_type = 'application/json')
+
+class CoreViewReceiveOutOfBandTest(TestCase):
+
+    @freeze_time("2017-11-17 10:00:00")
+    def setUp(self):
+        self.client = Client()
+        self.public_key = '85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2UssnPrCjHKTi5susO3wrr/q07eswumbL82b4HgOw=='
+        self.force_data = {
+            "type":      "MessageForceReportComputedTask",
+            "timestamp": 1510911047,  # 2017-11-17 9:30
+            "message_task_to_compute": {
+                "type":               "MessageTaskToCompute",
+                "timestamp":          1510909200,  # 2017-11-17 9:00
+                "task_id":            1,
+                "deadline":           1510915047  # 2017-11-17 10:37
+            }
+        }
+        message_timestamp   = datetime.datetime.now(timezone.utc)
+        new_message         = Message(
```

Comment:
Run model validations and create both objects in a single call. See remarks for `save_messages_to_database()`.